.github/buildomat/jobs/build-and-test.sh

```diff
@@ -28,12 +28,10 @@ rustc --version
 #   run.  Building with `--locked` ensures that the checked-in Cargo.lock
 #   is up to date.
 #
-# - Work-around for cargo#9895 via RUSTDOCFLAGS also.
-#
 banner build
 export RUSTFLAGS="-D warnings"
-export RUSTDOCFLAGS="-D warnings -Clink-args=-Wl,-R$(pg_config --libdir)"
-ptime -m cargo +'nightly-2021-09-03' build --locked --all-targets --verbose
+export RUSTDOCFLAGS="-D warnings"
+ptime -m cargo +'nightly-2021-11-24' build --locked --all-targets --verbose
```

Comment:
shouldn't cargo figured out the right cargo to run based on the rust toolchain file? i.e. shouldn't we not need to specify this?

Reply:
Yeah, I don't love that this is redundant here, and I don't fully understand why.  (That's true for a few things in the buildomat script.)